moveos/moveos-stdlib/moveos-stdlib/sources/object.move

```diff
@@ -150,8 +112,10 @@ module moveos_std::object {
     /// This function is only can be called by the module of `T`.
     public fun remove<T: key>(self: Object<T>) : T {
         let Object{id} = self;
+        // TODO Need to ensure that the Table is empty before delete the Object?
```

Comment:
Yes

Reply:
`destroy_empty_table` in raw_table already check if the table is empty